src/main/java/net/coderbot/iris/uniforms/CameraUniforms.java

```diff
@@ -40,10 +40,11 @@ static class CameraPositionTracker {
 		/**
 		 * Value range of cameraPosition. We want this to be small enough that precision is maintained when we convert
 		 * from a double to a float, but big enough that shifts happen infrequently, since each shift corresponds with
-		 * a noticeable change in shader animations and similar. 1024 is a reasonable compromise, since this means that
-		 * a player has to walk 1024 blocks on any axis in order to see a shift while playing.
+		 * a noticeable change in shader animations and similar. 1000024 is the number used by Optifine for walking (however this is too much, so we choose 30000),
+		 * with an extra 1024 check for if the user has teleported between camera positions.
 		 */
-		private static final double RANGE = 1024.0;
+		private static final double walkRange = 30000;
+		private static final double tpRange = 1000;
```

Comment:
Code style: In Java, it's convention for constants to be UPPER_CAMEL_CASE.

Reply:
Fixed

Reply:
🚲 🏠